apps/backend/admin-panel/src/modules/banner-types/banner-types.controller.ts

```diff
@@ -140,4 +141,19 @@ export class BannerTypesController {
       message: 'Banner type deleted successfully',
     };
   }
+
+  @Get('by-role/:role')
```

Comment:
`by-` is not required as it is it self explanatory 
```suggestion
  @Get('role/:role')
```